### PR TITLE
Changed type of variable "size"

### PR DESCRIPTION
**Pull request by vrbagalkote**
Varibale "size" type is changed from str to int
as I have to use it for arthimatic operations

Signed-off-by: Venkat R B <vrbagal1@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by vrbagalkote** [comment]
[job.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/487132/job.txt)

Attached is the job.log file.

File: io/disk/ssd/blkdiscard.py [hunk 1]
```diff
@@ -49,14 +52,13 @@ def test(self):
...
+        cmd = "blkdiscard %s -o 0 -v -l %d" % (self.disk, size)
         process.run(cmd, shell=True)
-        cmd = "blkdiscard %s -o %s -v -l %s" % (self.disk, size, size)
+        cmd = "blkdiscard %s -o %d \
+               -v -l %d" % (self.disk, size, size)
         process.run(cmd, shell=True)
-        cmd = "blkdiscard %s -o 0 -v -l %s" % (self.disk, size)
-        process.run(cmd, shell=True)
-        cmd = "blkdiscard %s -o %s -v -l 0" % (self.disk, size)
+        cmd = "blkdiscard %s -o %d -v -l 0" % (self.disk, size)
```
Comment (PraveenPenguin) [inline io/disk/ssd/blkdiscard.py]: can this be done in a helper function ?

**Comment by vrbagalkote** [comment]
Can we do as below?

process.run("blkdiscard %s -o 0 -v -l %d" % (self.disk, size), shell=True)
process.run("blkdiscard %s -o %d -v -l %d" % (self.disk, size, size), shell=True)
process.run("blkdiscard %s -o %d -v -l 0" % (self.disk, size), shell=True)

**Comment by PraveenPenguin** [comment]
it's a suggestion my point is can we remove repetitive code

**Pull request merged**